code/modules/mob/living/carbon/slime/life.dm

```diff
@@ -197,7 +197,7 @@
 		src.lying = 1
 		src.blinded = 1
 	else
-		if(src.paralysis || src.stunned || src.weakened || (status_flags && FAKEDEATH)) //Stunned etc.
+		if(src.paralysis || src.stunned || src.weakened || (has_trait(TRAIT_FAKEDEATH))) //Stunned etc.
```

Comment:
Does the `has_trait()` call actually need to be wrapped in brackets here? :thinking:

Reply:
Removed